### ISSUE TITLE
Storing the exchange

### ISSUE THREAD

**Issue by lorrp1** [mzheravin/exchange-core#84]
Is there a way to store "exchangeCore" on a database? 

I was thinking about using Redis but it can’t save custom data structures

**Comment by mzheravin** [comment]
Yes you just need to write implement ISerializationProcessor interface that will write into database and read from it.
If you only use orderbooks (no accounting and risk processors) - it is possible to track all incoming GCT (pending) orders and handle responses from exchange code to cleanup or update sizes/prices accordingly. In that case you just need to read pending orders from database during sartup and place them back to the order books.
